src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,10 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+  if (boolean === true) {
```

Comment:
Really good stuff. Challenge, can you refactor it using a ternary? I often use ternaries when I have a case of only two simple things to return